packages/framework/src/Model/Product/ProductVariantFacade.php

```diff
@@ -39,29 +35,21 @@ public function __construct(
      */
     public function createVariant(Product $mainProduct, array $variants)
     {
+        // parent method is not called intentionally
```

Comment:
there is no parent class :slightly_smiling_face:

Reply:
btw, do we still need the extended `ProductVariantFacade` in the project base? there is still some logic but I am not able to decode what is its real purpose

Reply:
implementation moved to framework